iac/example.yml

```diff
@@ -0,0 +1,33 @@
+# A sample workflow which checks out your Infrastructure-as-Code Configuration files,
+# such as Kubernetes, Helm & Terraform and scans them for any security issues.
+# The results are then uploaded to GitHub Security Code Scanning
+#
+# For more examples, including how to limit scans to only high-severity
+# issues, monitor images for newly disclosed vulnerabilities in Snyk and
+# fail PR checks for new vulnerabilities, see https://github.com/snyk/actions/
```

Comment:
Do we want to say vulnerabilities or stick with misconfigurations for IaC?

Reply:
i've stuck to saying "security issues" or "issues" here. Given the focus on security i think this makes sense in this context.